en/middleware/micro_dds_edits.md

```diff
@@ -0,0 +1,377 @@
+# MicroDDS
+
+The *PX4-Fast RTPS(DDS) Bridge*, which was also referred to as as the *microRTPS Bridge*, has now been replaced with MicroDDS bridge.
+
+The new approach provides a faster and simpler method of integrating applictaions running and linked in DDS domains (including ROS nodes), making it easy to share sensor data, commands, and other vehicle information.
+
+The following guide describes the new PX4 bridge architecture, and shows the reader how to write a simple *Micro DDS* application to subscribe * publish telemetry updates from the PX4 Autopilot.
+
+:::note
+Micro XRCE-DDS is a software solution that enables communictaion with an existing DDS network. 
+
+:::
```

Comment:
YOu might want to completely invert this, starting from the sentence you currently have in the note:

> Micro XRCE-DDS is a software solution that enables communication with an existing DDS network. 

 and ending with a sentence like the one you currently have first:
 
 > The  MicroDDS bridge replaces and earlier solution, the *PX4-Fast RTPS(DDS) Bridge*.
> The new approach provides a faster and simpler method of integrating applictaions running and linked in DDS domains (including ROS nodes), making it easy to share sensor data, commands, and other vehicle information.
  blah blah


Why? It is best to say what this document is about. Right now the fact it is replacing fastRTPS is interesting, but we won't care after the next release. We just say what it is, and add the fact it is better as a comment or a section further down.

Reply:
Noted. 
Btw I understand I caused a confusion, 
en/middleware/microdds.md is the most updated doc. 

I'll be sure to not create copies next time.

Reply:
No worries. I've moved things around to good order. COntent now needs to be written following that pattern.